runtime/syntax/nftables.yaml

```diff
@@ -6,14 +6,23 @@ detect:
     
 rules:
     - type: "\\b(chain|counter|map|rule|ruleset|set|table)\\b"
-    - type: "\\b(ether|icmp|icmpv6|icmpx|inet|ip|ip6|ipv4|ipv6|tcp|udp)\\b"
+    - type: "\\b(ether|inet|i(cm)?p(x|(v?(4|6))?)|tcp|udp)\\b"
     - special: "\\b(elements|hook|policy|priority|type)\\b"
     - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th)\\b"
     - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject)\\b"
     - preproc: "\\b(add|define|flush|include|delete)\\b"
-    - symbol: "[-=/:;,@]"
+    - symbol: "([-=/:;,@]|[!=])"
     - symbol.operator: "[<>.&|^!]|\\b(and|ge|gt|le|lt|or|xor)\\b"
```

Comment:
`==` and `!=` are comparison operators, so why not adding them to `symbol.operator`? I even suggest to do the same with `-` and `=`.

Example (from C):
https://github.com/zyedidia/micro/blob/b3227d60498938a06d16d9ea405417ce70a52158/runtime/syntax/c.yaml#L23

Reply:
That was a Suggestion on the old PR. I will change it later. It will make more sense to add it to the operators

Reply:
I only gave examples. Never mind...
We should add all operators from `symbol` to `symbol.operator`, like `-`, `+`, `*`, `/`, etc.

Reply:
I am working right now on it, should i move the symbols `=`, `:`, `;`, `,` and `@` also to symbol.operator? I have now added `-`, `+`, `*`, `/`.

Reply:
Okay I`ve added it and some additional changes like more filetypes, removed spaces at the end.